proto/viam/app/packages/v1/packages.proto

```diff
@@ -81,6 +81,7 @@ message Package {
   PackageInfo info = 1;
   string url = 2;
   google.protobuf.Timestamp created_on = 3;
+  string checksum = 4;
```

Comment:
FYI: I think this should match the GCS CRC32C that will be available. See: https://cloud.google.com/storage/docs/hashes-etags